psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -562,7 +628,7 @@ void FISAPT::nuclear() {
 
     matrices_["VC"] = std::make_shared<Matrix>("VC", nm, nm);
     Vint->compute(matrices_["VC"]);
-
+ 
```

Comment:
Seems there's a bit of extra whitespace to get rid of here.

Reply:
Fixed.